---
title: "Bioenergetics and Quinones"
excerpt: "We study the evolution of metabolic pathways involved in bioenergetics.<br/><img src='/images/quinone-species-1b.png' width='300' alt='Quinone molecules'>"
collection: projects
---

We study the origins and diversification of metabolic pathways involved in bioenergetics using phylogenomics and experimental approaches.

We investigate in particular, the **evolution of the different biosynthetic pathways of quinones** (find our recent [review here](https://doi.org/10.1016/j.bbabio.2020.148259)). Quinones are molecules involved in electron transfer chains. They are therefore crucial for cellular energy production.

<img src='/images/UQ-pw.png' width='800'>

*Variability of the ubiquinone biosynthetic pathway. In particular, three hydroxylation steps can be performed by different sets of enzymes depending on the species. In purple, UbiM performs the three hydroxylation steps (as in Neisseria meningitidis). In blue, two enzymes are required, as in Rhodobacter capsulatus. In red, three enzymes are required, as in Escherichia coli. Figure from [Kazemzadeh et al. Mol. Biol. Evol, 2023](https://doi.org/10.1093/molbev/msad219).*

We combine **experimental and computational approaches**. This goes from developing tools for genome annotation of the proteins involved in the pathways, to the study of their functional diversification, and *in vivo* testing of their predicted functions. 

## The QuinEvol project

An example of study that combines **experimental and computational approaches** was recently [published](https://doi.org/10.1093/molbev/msad219). We show the diversity of the pathway used by *Pseudomonadota* (ex-proteobacteria) to produce ubiquinone, and at the same time explore the diversification of the flavin monooxygenase (FMO) protein family by gene transfers and duplications. 

<img src='/images/evol-UQ-FMO-mbe.png' width='600'>

*Evolution and diversification of the UQ-FMO protein family. Members of this family of hydroxylases evolved by duplication different regio-selectivities within the ubiquinone biosynthetic pathway - being able to hydroxylase Carbon 1, 5 and/or 6 of the UQ precursor. Figure from [Kazemzadeh et al. Mol. Biol. Evol, 2023](https://doi.org/10.1093/molbev/msad219).*

Team members also recently characterized [a new pathway for ubiquinone production](https://journals.asm.org/doi/full/10.1128/mBio.01319-19), which is O2-independent (contrary to the classical, O2-dependent one). We could recently contribute to understanding its origin and physiological role. 

We recently [published a study](https://doi.org/10.1093/ismejo/wrae253) on the repertoire of quinones found in *Pseudomonadota* and how the dynamics of this repertoire likely accompagnied the diversification of aerobic, facultative aerobic, microaerophilic and anaerobic metabolisms. 
To this end, we proposed an evolutionary scenario to explain the variable content in quinones across *Pseudomonadota*. 
<img src='/images/ISMEJ_wrae253f6_red.jpeg' width='800'>

*Global evolutionary scenario for the quinone repertoire of Pseudomonadota.* For full legend see [here](https://academic.oup.com/view-large/figure/499658360/wrae253f6.tif).

Finally, we could show that *Cyanobacteria* inherited from *Pseudomonadota* an enzyme that is crucial for the production of the plastoquinone (PQ), used in photosynthetic chains. This enzyme is a flavin monooxygenase, and we show it requires O2 to proceed. Hence, most contemporary *Cyanobacteria* need O2 for oxygenic photosynthesis. This work was recently published at **[the FEBS Journal](https://doi.org/10.1111/febs.70272)**. 


*The [QuinEvol project](https://anr.fr/Projet-ANR-21-CE02-0018) was funded by the ANR (2021-2025).*

## The AbiSYM project

Now that we understand well the evolution of the ubiquinone biosynthetic pathway in *Pseudomonadota*, we are currently investigating how comes that eukaryotes did not inherit in their mitochondria, the exact same UQ pathway that is found in the lineages that are close to the ancestor of the mitochondrion. Answering this question might bring light on the establishment of bioenergetic processes during eukaryogenesis, and in particular, the potential contribution of other prokaryotic contributors than *Pseudomonadota*. 


*The [AbiSYM project](https://anr.fr/Project-ANR-23-CE02-0016) is funded by the [ANR](https://anr.fr/) (2024-2029).*

## The ToLQuin project

We recently contributed to a study that reports the **discovery of a novel quinone**, the methyl-plastoquinone, in the phylum *Nitrospirota*. 
This was [published in *PNAS* in 2025](https://doi.org/10.1073/pnas.2421994122). 

In a newly funded project, together with our partner Dr. Felix Elling (U. Kiel, Germany), we will investigate the **possibility of novel quinones** in the tree of life, explore **lowly characterized quinone pathways**, and investigate the impact of quinone evolution on the **diversification of energy metabolism**. 

*The ToLQuin project is funded by the [ANR](https://anr.fr/) and the [DFG](https://www.dfg.de/en) (2026-2029).*

## The ADAPT2Q project

In collaboration with the [BIP](https://bip.cnrs.fr/) and the [LCB](https://lcb.cnrs.fr/) labs in Marseille, we are currently working on the [ADAPT2Q](https://anr.fr/Projet-ANR-23-CE44-0012) ANR-funded project that aims at understanding the adaptation of quinone-interacting enzymes to the diversity of quinones that have evolved along time. 

*The [ADAPT2Q project](https://anr.fr/Projet-ANR-23-CE44-0012) is funded by the ANR (2023-2027).*

### Funders

These projects are mainly funded by the French National Research Agency, ["ANR"](https://anr.fr/).

<img src='/images/ANR-logo-2021-sigle.jpg' width='100' alt='ANR logo' style='float: left; margin-right: 10px;'/>
